sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableAsyncClientTest.java

```diff
@@ -67,16 +67,18 @@ protected HttpClient buildAssertingClient(HttpClient httpClient) {
 
     protected void beforeTest() {
         final String tableName = testResourceNamer.randomName("tableName", 20);
-        tableClient = getClientBuilder(tableName, true).buildAsyncClient();
+        tableClient = getClientBuilder(tableName, false)
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
```

Comment:
Was this line left by accident after testing?

Reply:
@vcolin7 Yes this was added for debugging and I forgot remove it.